### PR TITLE
bug fixes and improvements

### PR DESCRIPTION
**Pull request by aditya-tezsure**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,146 @@
-## This project was generated with React.Js & Electron.
+# Tezster GUI
 
-Tezster-GUI comes with pretty much the same functionalities as CLI but in its beta version, it comes integrated with the CLI. After starting tezos node on your local machine with Tezster-CLI, you can clone it from here.
-To start the app in the dev environment run the following command :
+<img src="https://tezster.s3-ap-southeast-1.amazonaws.com/TEZSTER_GUI/1_jDB5enULQVo2UfeiwD32qA.png" alt="tezster GUI banner" align="center" />
+<br />
+<div align="center"><strong>Start interacting with Tezos Chain within seconds</strong></div>
+<div align="center">Use this GUI version of Tezster which is ready to use Desktop app, made with React and Electron. Primarily can be used to interact with <a href="https://docs.tezster.tech/tezster-cli">Tezster cli</a>  ( i.e Local Node ) as well as remote Testnets. The below documentation will help you get started with Tezster-GUI although to get complete understanding of components usage and visual demo follow the link <a href="https://docs.tezster.tech/">https://docs.tezster.tech/</a>
```
Comment (jvtezsure) [inline README.md:6]: Use this GUI version of Tezster which is ready-to-use Desktop app, made with React and Electron. It allows you to interact with Tezos blockchain and to deploy/test smart contracts on Tezos testnets or Sandbox node (which requires Tezster-CLI). 


update the desccription accordigly.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,146 @@
...
+    curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+    sudo apt-get install nodejs
+
+After installing verify and check the installed version.
+
+    node -v
+
+_Note: Please make sure you have the latest npm and node.js versions to be 12.x or above._
+
+_Note: Please make sure you have the latest npm and node.js versions to be 12.x or above._
```
Comment (jvtezsure) [inline README.md:42]: Duplicate line.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,146 @@
...
+
+    tezster start-nodes
+
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+    git clone https://github.com/Tezsure/Tezster-GUI.git
+
+Now the second step is to navigate to the downloaded directory using the command.
```
Comment (jvtezsure) [inline README.md:64]: You dont need to explain the 'cd /tezter-gui' step and all 
You can combine all steps together. 

For npm run dev, you can have a separate step.

File: README.md [hunk 4]
```diff
@@ -1,22 +1,146 @@
...
+
+    cd /Tezster-GUI
+
+‌
+The third step is to download the required installation packages using the command
+
+    sudo npm i --save
```
Comment (jvtezsure) [inline README.md:71]: Why do we need sudo?
Why do we need to use --save command?

Comment (aditya-tezsure) [reply README.md:71]: some of the electron binary files need system permission to install

File: .env [hunk 5]
```diff
@@ -0,0 +1,3 @@
+localNodeAddress=http://localhost:18731/
+testNodeAddress=https://tezos-dev.cryptonomic-infra.tech/
```
Comment (jvtezsure) [inline .env:2]: What is testNodeAddress?

File: README.md [hunk 6]
```diff
@@ -1,22 +1,151 @@
...
+<img  src="https://tezster.s3-ap-southeast-1.amazonaws.com/TEZSTER_GUI/1_jDB5enULQVo2UfeiwD32qA.png"  alt="tezster GUI banner" align="center" />
 
-```bash
-$ npm run dev
-```
+<br />
+
+<div align="center"><strong>Start interacting with Tezos Chain within seconds</strong></div>
+
+<div align="center">Use this GUI version of Tezster which is ready-to-use Desktop app, made with React and Electron. It allows you to interact with Tezos blockchain and to deploy/test smart contracts on Tezos testnets or Sandbox node (which requires <a  href="https://docs.tezster.tech/tezster-cli">Tezster-CLI</a>).
```
Comment (jvtezsure) [inline README.md:9]: This is not appropriate description. Previous one had was more detailed. Where's the link for GUI documentation.

File: README.md (hunk 2 above)
Comment (jvtezsure) [reply README.md:49]: You need to resolve all the comments once you've made the change.

File: README.md [hunk 7]
```diff
@@ -1,22 +1,151 @@
...
+sudo npm install -g tezster@latest
+
+Run Following to ensure the installed version of [Tezster-CLI](https://docs.tezster.tech/tezster-cli).
+
+tezster --version
+
+To start the tester nodes on your machine run:
+
+tezster start-nodes
+
```
Comment (jvtezsure) [inline README.md:64]: User a code-block to format commands. 
This applies to whole readme file.

File: README.md [hunk 8]
```diff
@@ -1,22 +1,151 @@
...
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+git clone https://github.com/Tezsure/Tezster-GUI.git
+
+Now the second step is to navigate to the downloaded directory and download the required installation packages using the command
+
+sudo npm i --save
```
Comment (jvtezsure) [inline README.md:73]: why do i need to run npm with sudo 
and also why do I need to run it with --save flag?

I believe I'm commenting this 2nd time.

File: .env (hunk 5 above)
Comment (aditya-tezsure) [reply .env]: no need of env file anymore moved everything to config file just like cli

File: README.md (hunk 8 above)
Comment (aditya-tezsure) [reply README.md]: electron works on top of gcc compiler which in linux and mac operating system requires access to .bin folder to install required binary files on the machine which is a system file and it requires root access.

File: README.md [hunk 9]
```diff
@@ -1,22 +1,146 @@
...
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+    git clone https://github.com/Tezsure/Tezster-GUI.git
+
+Download the required installation packages using the command
+
+    sudo npm install
+
```
Comment (jvtezsure) [inline README.md:71]: can we make it simple like this? 
image

File: package.json [hunk 10]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tezster-gui-v2.0",
+  "name": "tezster-gui",
   "productName": "Tezster-GUI",
-  "version": "2.0.0",
-  "description": "Tezster-GUI",
+  "version": "2.1.0",
+  "description": "GUI version of Tezster which is ready to use Desktop app, made with React and Electron",
```
Comment (jvtezsure) [inline package.json:5]: Tezster GUI is a desktop app to interact & develop/deploy smart contracts on Tezos blockchain. 

you can consult Om for better one line description.

File: package.json [hunk 11]
```diff
@@ -1,8 +1,8 @@
...
   "scripts": {
     "build": "concurrently \"yarn build-main\" \"yarn build-renderer\"",
```
Comment (jvtezsure) [inline package.json:7]: change 'appId' for electron to a relevant appId.

File: package.json [hunk 12]
```diff
@@ -100,20 +100,16 @@
     },
     "publish": {
       "provider": "github",
-      "owner": "TEZSURE GUI V2.0",
-      "repo": "TEZSURE GUI V2.0",
+      "owner": "TEZSURE GUI",
```
Comment (jvtezsure) [inline package.json:103]: Owner is 'Tezsure Inc' 
please make this change wherever required. 
also change 'repo' to appropriate value.

File: app/components/Error.js [hunk 13]
```diff
@@ -9,11 +13,39 @@ function Error() {
...
+              {props.isAvailableTezsterCli === 'pending'
+                ? 'Please wait...'
+                : 'Sorry'}
+            </b>
           </div>
           <p className="workplace-description">
-            Please install Tezster-CLI and restart the application
+            {props.isAvailableTezsterCli === 'pending'
+              ? 'While we check whether tezster-cli is running'
+              : 'Please install Tezster-CLI to use the localnode \n click on button below to continue with Carthagenet'}
```
[the remark this answers is not in the document]
Comment (aditya-tezsure) [reply app/components/Error.js]: tezster logo is svg it can't be increased I'll ask sanket to give me bigger image

File: package.json (hunk 11 above)
Comment (aditya-tezsure) [reply package.json:7]: what shall I put as appId can you please suggest

File: app/components/Error.js (hunk 13 above)
[the remark this answers is not in the document]
Comment (jvtezsure) [reply app/components/Error.js]: Can't we scale using css? I know it'll be stretched but we can enlarge till it looks good. 
also pls ask sanket for bigger logo.

[the remark this answers is not in the document]
Comment (aditya-tezsure) [reply app/components/Error.js]: done

File: package.json (hunk 11 above)
Comment (aditya-tezsure) [reply package.json:7]: "appId": "com.electron.tezster-gui" I have used as an app id

File: README.md [hunk 14]
```diff
@@ -1,22 +1,144 @@
...
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+    git clone https://github.com/Tezsure/Tezster-GUI.git
+    cd Tezster-GUI/
+    sudo npm install
+
+‌
+
```
Comment (jvtezsure) [inline README.md:71]: Extra white space.

File: README.md [hunk 15]
```diff
@@ -1,22 +1,144 @@
...
 To create build package for other platform run the following command
 
-```bash
-$ npm run package-all
-```
+    npm run package-all
+
+On linux systems it requires an additional dependency of wine package installer to create a build package for windows environment.‌
+
+## Steps to install wine on linux
```
Comment (jvtezsure) [inline README.md:94]: This requires smaller font-size as it's a sub-header under 'Building packages for all platforms'

**Pull request merged**